test/e2e/data/infrastructure-docker/cluster-template.yaml

```diff
@@ -1,147 +1,139 @@
-apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
-kind: DockerCluster
+apiVersion: v1
```

Comment:
I would like to gitignore the generated cluster-templates and keep under source control only the bases, but it seems that this makes the apidiff batch to fail 🤔